torch/nn/qat/modules/embedding_ops.py

```diff
@@ -61,7 +127,6 @@ def from_float(cls, mod):
         qat_embedding_bag = cls(mod.num_embeddings, mod.embedding_dim, mod.max_norm, mod.norm_type,
                                 mod.scale_grad_by_freq, mod.mode, mod.sparse, mod.weight,
                                 mod.include_last_offset, mod.padding_idx, qconfig=qconfig)
-        qat_embedding_bag.weight = mod.weight
```

Comment:
why did we remove this line?

Reply:
This is not required as mod.weight is being passed as a parameter to the constructor.